internal/tools/aws/database.go

```diff
@@ -41,11 +46,45 @@ func (d *RDSDatabase) Provision(store model.InstallationDatabaseStoreInterface,
 
 // Teardown removes all AWS resources related to a RDS database.
 func (d *RDSDatabase) Teardown(keepData bool, logger log.FieldLogger) error {
-	err := rdsDatabaseTeardown(d.installationID, keepData, logger)
+	logger.Info("Tearing down AWS RDS database")
+	awsID := CloudID(d.installationID)
+
```

Comment:
add:
`logger = logger.WithField("db-cluster-name", awsID)` here and then remove the WithField portion on lines 62 and 64 for cleanup and additional context when `logger ` is passed to other functions on lines 52 and 58